### ISSUE TITLE
Notification of new comments on application

### ISSUE THREAD

**Issue by erinlessard** [tyler274/Recruitment-App#22]

**Comment by tyler274** [comment]
Doing this would require some ajax and javascript as flask uses the Request/Response lifecycle for user interaction.

It's a bit beyond my current capability, not impossible, and not necessarily difficult but not something I want to deal with at the moment. 

Unless of course you mean an email to everyone who has commented, that's possible.

**Comment by erinlessard** [comment]
yeah, an email just to all recruiters would probably be ok. not sure if we want/need this yet, depends on how our workflow goes in the coming week, let's keep it open.
